http_check/check.py

```diff
@@ -319,6 +320,14 @@ def send_status_down(loginfo, message):
             else:
                 send_status_up("%s is UP" % addr)
 
+        # Report status as a metric as well
+        can_status = 1 if service_checks[0][1] == "UP" else 0
```

Comment:
Grabbing it in this manner is going to be error prone. Please set them in the same way as the service checks themselves.